app/assets/javascripts/static_pages.js

```diff
@@ -0,0 +1,16 @@
+//= require medium-editor
+
+WebsiteOne.define('StaticPages', function() {
+    return {
+        init: function() {
+            var editor = new MediumEditor('.editable', {
+                placeholder: {
+                    /* This example includes the default options for placeholder,
+                        if nothing is passed this is what it used */
+                    text: 'Type your text',
+                    hideOnClick: true
+                }
+            }
+        };
+    }
+});
```

Comment:
The medium rails gem does not specify.  I'm thinking of trying this approach: http://brandonhilkert.com/blog/page-specific-javascript-in-rails/